pkg/acs/calacs/acsccd/doccd.c

```diff
@@ -438,6 +439,10 @@ int DoCCD (ACSInfo *acs_info) {
         if (acs_info->printtime)
             TimeStamp("SINKCORR complete", acs_info->rootname);
     }
+
+    if (!OmitStep(acs[0].sinkcorr))
```

Comment:
I would have also changed line 431 ``if (acs->sinkcorr == PERFORM)`` to call ``OmitStep()``, which should probably be made inline also. Given the hastiness for this fix, leave this as is.